### PR TITLE
Patch Android for release

### PR DESCRIPTION
**Pull request by henrytao-me**
Follow up https://github.com/CovidShield/mobile/pull/164

### CODE REVIEW

**Comment by henrytao-me** [comment]
Tested again on Android. ✅

File: package.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
     "react-native-snap-carousel": "^3.9.0",
     "react-native-splash-screen": "^3.2.0",
     "react-native-svg": "^12.1.0",
-    "react-native-system-setting": "^1.7.4",
+    "react-native-system-setting": "1.7.4",
```
Comment (henrytao-me) [inline package.json:52]: Set to fix version just to make sure the patch works on exact version.

File: android/build.gradle [hunk 2]
```diff
@@ -3,7 +3,7 @@
 buildscript {
     ext {
         buildToolsVersion= "28.0.3"
-        minSdkVersion = 21
+        minSdkVersion = 23
```
Comment (henrytao-me) [inline android/build.gradle:6]: Requirement for Android after Google Play submission review.

**Comment by gavrix** [comment]
I believe only location-related code would have been enough

**Comment by henrytao-me** [comment]
I think I can remove everything just in case.

**Pull request merged**